common/utils/cstor_pool_deploy_check.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This utility checks if all the pool pods are running and are scheduled on different nodes.
+  
+- name: Obtain the number of CSP.
+  shell: kubectl get csp --no-headers | wc -l
+  args:
+    executable: /bin/bash
+  register: csp_count
+  
+- name: Checking all the pool pods are in running state  
+  shell: kubectl get po -l app=cstor-pool -n openebs -o jsonpath='{.items[?(@.status.phase=="Running")].metadata.name}' | wc -w
```

Comment:
@shashank855  can you use vars instead of using "openebs" as a namespace here

Reply:
@nsathyaseelan I have made the changes !!